Traceback (most recent call last):
  File "C:\Users\mshem\anaconda3\envs\mshemuni.github.io\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\mshem\anaconda3\envs\mshemuni.github.io\lib\site-packages\nbclient\client.py", line 1117, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\mshem\anaconda3\envs\mshemuni.github.io\lib\site-packages\nbclient\util.py", line 78, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\mshem\anaconda3\envs\mshemuni.github.io\lib\site-packages\nbclient\util.py", line 57, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\mshem\anaconda3\envs\mshemuni.github.io\lib\asyncio\base_events.py", line 642, in run_until_complete
    return future.result()
  File "C:\Users\mshem\anaconda3\envs\mshemuni.github.io\lib\site-packages\nbclient\client.py", line 558, in async_execute
    await self.async_execute_cell(
  File "C:\Users\mshem\anaconda3\envs\mshemuni.github.io\lib\site-packages\nbclient\client.py", line 862, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "C:\Users\mshem\anaconda3\envs\mshemuni.github.io\lib\site-packages\nbclient\client.py", line 765, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
dct = {"ad": "Muhammed", "soyad": "Shameoni Niaei", "dtarihi": 1988}

dct["adi"]
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mKeyError[0m                                  Traceback (most recent call last)
[1;32m~\AppData\Local\Temp/ipykernel_21988/1509176710.py[0m in [0;36m<module>[1;34m[0m
[0;32m      1[0m [0mdct[0m [1;33m=[0m [1;33m{[0m[1;34m"ad"[0m[1;33m:[0m [1;34m"Muhammed"[0m[1;33m,[0m [1;34m"soyad"[0m[1;33m:[0m [1;34m"Shameoni Niaei"[0m[1;33m,[0m [1;34m"dtarihi"[0m[1;33m:[0m [1;36m1988[0m[1;33m}[0m[1;33m[0m[1;33m[0m[0m
[0;32m      2[0m [1;33m[0m[0m
[1;32m----> 3[1;33m [0mdct[0m[1;33m[[0m[1;34m"adi"[0m[1;33m][0m[1;33m[0m[1;33m[0m[0m
[0m
[1;31mKeyError[0m: 'adi'
KeyError: 'adi'

